src/ui.rs

```diff
@@ -43,6 +43,11 @@ struct ConsoleState {
     message_count: usize,
 }
 
+pub struct MainMenuBarStatus {
```

Comment:
:+1:  we should probably be using these return types instead of `&mut` parameters for the gui when we want multiple return values.